Source/WebKit/UIProcess/Cocoa/WKShareSheet.mm

```diff
@@ -108,7 +141,17 @@ static void appendFilesAsShareableURLs(RetainPtr<NSMutableArray>&& shareDataArra
                 shareDataArray = nullptr;
                 break;
             }
+
+#if PLATFORM(IOS_FAMILY)
+            RetainPtr itemProvider = adoptNS([[WKShareSheetFileItemProvider alloc] initWithURL:fileURL]);
+            if (!itemProvider) {
+                shareDataArray = nullptr;
```

Comment:
Nit - this is consistent with the above, but I wonder if we should change these to `nil` instead of `nullptr`.